### PR TITLE
#163: Convert column names in auto inference

### LINKED ISSUE #163: Auto Inference should keep original column names
See issue https://github.com/exasol/virtual-schema-common-document/issues/189:

> Auto inference should have an option keep the original column names instead of always changing them to UPPER_SNAKE_CASE. The conversion is only mentioned for CSV files, so it happening for Parquet files also may be a bug.

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #163

### CODE REVIEW

File: src/main/java/com/exasol/adapter/document/documentfetcher/files/parquet/ParquetColumnToMappingDefinitionConverter.java [hunk 1]
```diff
@@ -25,28 +27,39 @@ private static String buildColumnName(final Type column) {
     MappingDefinition convert(final Type column) {
         final LogicalTypeAnnotation logicalTypeAnnotation = column.getLogicalTypeAnnotation();
         if (logicalTypeAnnotation != null) {
-            return new LogicalTypeConverter().convert(logicalTypeAnnotation, buildColumnName(column));
+            return new LogicalTypeConverter().convert(logicalTypeAnnotation,
+                    columnNameConverter.convertColumnName(column.getName()));
```
Comment (tomuben) [inline src/main/java/com/exasol/adapter/document/documentfetcher/files/parquet/ParquetColumnToMappingDefinitionConverter.java:31]:
```suggestion
                    this.columnNameConverter.convertColumnName(column.getName()));
```

File: src/main/java/com/exasol/adapter/document/documentfetcher/files/parquet/ParquetColumnToMappingDefinitionConverter.java [hunk 2]
```diff
@@ -25,28 +27,39 @@ private static String buildColumnName(final Type column) {
...
+        }
+
         @Override
         public void visit(final GroupType groupType) {
             groupType.getLogicalTypeAnnotation();
             final Fields.FieldsBuilder fieldsBuilder = Fields.builder();
             for (final Type column : groupType.getFields()) {
-                final MappingDefinition columnMapping = new ParquetColumnToMappingDefinitionConverter().convert(column);
+                final MappingDefinition columnMapping = new ParquetColumnToMappingDefinitionConverter(
+                        columnNameConverter).convert(column);
```
Comment (tomuben) [inline src/main/java/com/exasol/adapter/document/documentfetcher/files/parquet/ParquetColumnToMappingDefinitionConverter.java:53]:
```suggestion
                        this.columnNameConverter).convert(column);
```

File: src/main/java/com/exasol/adapter/document/documentfetcher/files/parquet/ParquetColumnToMappingDefinitionConverter.java [hunk 3]
```diff
@@ -25,28 +27,39 @@ private static String buildColumnName(final Type column) {
...
                 fieldsBuilder.mapField(column.getName(), columnMapping);
             }
             this.result = fieldsBuilder.build();
         }
 
+        private String buildColumnName(final Type column) {
+            return columnNameConverter.convertColumnName(column.getName());
+        }
```
Comment (tomuben) [inline src/main/java/com/exasol/adapter/document/documentfetcher/files/parquet/ParquetColumnToMappingDefinitionConverter.java:61]: Is this needed? Can't find any reference.....

Comment (kaklakariada) [reply src/main/java/com/exasol/adapter/document/documentfetcher/files/parquet/ParquetColumnToMappingDefinitionConverter.java:61]: Yes. There was a static method with the same name. The method now needs the `columnNameConverter`, so I moved the method here.

**Review by tomuben — APPROVED** [review]

**Pull request merged**